### PR TITLE
Scale nginx default-backends by number of workers

### LINKED ISSUE #40: Scale ingress-nginx default backends
Based on number of workers.

### PR DESCRIPTION
**Pull request by kke**
Fixes #40

Something like this?

### CODE REVIEW

File: lib/pharos/addons/ingress_nginx.rb [hunk 1]
```diff
@@ -38,7 +38,8 @@ def install
         apply_stack(
           configmap: config.configmap || {},
           node_selector: config.node_selector || {},
-          image: image_name
+          image: image_name,
+          worker_hosts: addon_manager.config.worker_hosts.size
```
Comment (SpComb) [inline lib/pharos/addons/ingress_nginx.rb]: The addons already have `config.worker_hosts.size`, no need to pass the `addon_manager`

**Comment by jakolehm** [comment]
Issue description was not the best one. I meant that we should scale default backend to more than one replica. IMHO it does not make sense to scale this to every node (and then we should use daemonset).

Maybe something similar to kube-dns?

File: lib/pharos/addons/ingress_nginx.rb (hunk 1 above)
Comment (kke) [reply lib/pharos/addons/ingress_nginx.rb]: Addon's `config` is the struct/schema thing for the current addon:

```ruby
config = configs[addon_class.name]
```

Comment (SpComb) [reply lib/pharos/addons/ingress_nginx.rb]: Bleh, right. The addons should probably also get the top-level config... passing the `addon_manager` doesn't feel right.

**Pull request closed without merging**